pkg/kudoctl/cmd/env/environoment.go

```diff
@@ -0,0 +1,57 @@
+package env
```

Comment:
I think the current convention is that /cmd/xxx is reserved for commands. since env is not command I would place it one level up

Reply:
this env would be used for cmds only... I would not expect this environment to be used for the manager... or at least I didn't consider that.

Reply:
one level up from `cmd` is `kudoctl`... I'll move there.